app/src/main/res/layout/about_activity.xml

```diff
@@ -22,21 +22,400 @@
 
     </com.google.android.material.appbar.AppBarLayout>
 
-    <LinearLayout
+    <ScrollView
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
+        android:layout_height="match_parent"
         android:layout_marginTop="?attr/actionBarSize"
-        android:orientation="vertical"
         android:padding="10dp">
 
-        <TextView
-            android:id="@+id/aboutText"
+        <LinearLayout
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:linksClickable="true"
-            android:singleLine="false"
-            android:focusable="true" />
+            android:orientation="vertical">
+
+            <androidx.constraintlayout.widget.ConstraintLayout
+                android:id="@+id/version_history"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:padding="8dp">
+
+                <TextView
+                    android:id="@+id/tv1"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:fontFamily="sans-serif-medium"
+                    android:padding="2dp"
+                    android:text="@string/version_history"
+                    android:textColor="@android:color/black"
+                    android:textSize="18sp"
+                    app:layout_constraintLeft_toLeftOf="parent"
```

Comment:
This looks a bit broken on right-to-left languages. You should use `app:layout_constraintStart_toStartOf` and `app:layout_constraintEnd_toEndOf` instead of `app:layout_constraintLeft_toLeftOf` and `app:layout_constraintRight_toRightOf`. That way it is on the left for left-to-right languages but on the right for right-to-left languages.